x/batching/keeper/abci.go

```diff
@@ -74,44 +88,44 @@ func (k Keeper) ConstructBatch(ctx sdk.Context) (types.Batch, error) {
 // ConstructDataResultTree constructs a data result tree based on the
 // data results that have not been batched yet. It returns the tree's
 // entries (unhashed leaf contents) and hex-encoded root.
-func (k Keeper) ConstructDataResultTree(ctx sdk.Context) ([][]byte, string, error) {
+func (k Keeper) ConstructDataResultTree(ctx sdk.Context) ([][]byte, []byte, error) {
```

Comment:
Not something for this PR, just wanted to get the stupid idea out there. If there are no new DR results does it make sense to just reuse the previous DR results root? As a poor mans batch skip :P

Reply:
I'll make sure to address that point